components/scream/src/control/tests/sc_tests.cpp

```diff
@@ -262,6 +262,7 @@ TEST_CASE ("recreate_mct_coupling")
   FID sfc_alb_dir_nir_id  ("sfc_alb_dir_nir",    scalar2d_layout, nondim,  grid_name);
   FID sfc_alb_dif_vis_id  ("sfc_alb_dif_vis",    scalar2d_layout, nondim,  grid_name);
   FID sfc_alb_dif_nir_id  ("sfc_alb_dif_nir",    scalar2d_layout, nondim,  grid_name);
+  FID surf_lw_flux_up_id  ("surf_lw_flux_up",    scalar2d_layout, nondim,  grid_name);
```

Comment:
Just a note, I think there was some discussion about changing this to `sfc_lw_flux_up`.  Is that still what we want to do?  @brhillman ?

Reply:
Just noting for future, yes I think we want to be consistent in naming. Will fix in a subsequent PR.